install.sh

```diff
@@ -2,13 +2,23 @@
 
 # shellcheck disable=SC2164
 # shellcheck disable=SC2103
+declare -r BASHUNIT_GIT_REPO="https://github.com/TypedDevs/bashunit"
+
+function get_latest_tag() {
```

Comment:
Can you write a test to ensure the behaviour of this new function?

Reply:
I am using the existing function for the latest update; 
A test case already exists to test the function test_get_latest_tag() in \bashunit\tests\unit\helpers_test.sh.

Reply:
You are right, that test is covering that area as well, thanks!